msgid ""
msgstr ""
"Project-Id-Version: duplicity\n"
"Report-Msgid-Bugs-To: Kenneth Loafman <kenneth@loafman.com>\n"
"POT-Creation-Date: 2024-11-17 10:47-0600\n"
"PO-Revision-Date: 2024-11-17 17:08\n"
"Last-Translator: \n"
"Language-Team: Arabic\n"
"Language: ar_SA\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=6; plural=(n==0 ? 0 : n==1 ? 1 : n==2 ? 2 : n%100>=3 && n%100<=10 ? 3 : n%100>=11 && n%100<=99 ? 4 : 5);\n"
"X-Crowdin-Project: duplicity\n"
"X-Crowdin-Project-ID: 539508\n"
"X-Crowdin-Language: ar\n"
"X-Crowdin-File: /main/po/duplicity.pot\n"
"X-Crowdin-File-ID: 6\n"

#: ../duplicity/__main__.py:76
msgid "INT intercepted...exiting."
msgstr "تم اعتراض...الخروج."

#: ../duplicity/__main__.py:84
#, python-format
msgid "GPG error detail: %s"
msgstr "تفاصيل خطأ GPG: %s"

#: ../duplicity/__main__.py:91
#, python-format
msgid "User error detail: %s"
msgstr "تفاصيل خطأ المستخدم: %s"

#: ../duplicity/__main__.py:98
#, python-format
msgid "Backend error detail: %s"
msgstr "تفاصيل خطأ الخلفية: %s"

#: ../duplicity/argparse311.py:291
msgid "usage: "
msgstr "الاستخدام: "

#: ../duplicity/argparse311.py:761
#, python-format
msgid "argument %(argument_name)s: %(message)s"
msgstr "الحجة %(argument_name)s: %(message)s"

#: ../duplicity/argparse311.py:870
msgid ".__call__() not defined"
msgstr ". لم يتم تعريف__call__()"

#: ../duplicity/argparse311.py:1115
#, python-format
msgid "conflicting subparser: %s"
msgstr "فصل فرعي متعارض: %s"

#: ../duplicity/argparse311.py:1118
#, python-format
msgid "conflicting subparser alias: %s"
msgstr "conflicting subparser alias: %s"

#: ../duplicity/argparse311.py:1152
#, python-format
msgid "unknown parser %(parser_name)r (choices: %(choices)s)"
msgstr "محلل غير معروف %(parser_name)r (خيارات: %(choices)s)"

#: ../duplicity/argparse311.py:1215
#, python-format
msgid "argument \"-\" with mode %r"
msgstr ""

#: ../duplicity/argparse311.py:1223
#, python-format
msgid "can't open '%(filename)s': %(error)s"
msgstr "لا يمكن فتح '%(filename)s': %(error)s"

#: ../duplicity/argparse311.py:1427
#, python-format
msgid "cannot merge actions - two groups are named %r"
msgstr "لا يمكن دمج الإجراءات - مجموعتان تسميهما %r"

#: ../duplicity/argparse311.py:1462
msgid "'required' is an invalid argument for positionals"
msgstr "'مطلوب' هو حجة غير صالحة للموقعين"

#: ../duplicity/argparse311.py:1483
#, python-format
msgid "invalid option string %(option)r: must start with a character %(prefix_chars)r"
msgstr "سلسلة خيار غير صالحة %(option)r: يجب أن تبدأ بحرف %(prefix_chars)r"

#: ../duplicity/argparse311.py:1500
#, python-format
msgid "dest= is required for options like %r"
msgstr ""

#: ../duplicity/argparse311.py:1517
#, python-format
msgid "invalid conflict_resolution value: %r"
msgstr ""

#: ../duplicity/argparse311.py:1534
#, python-format
msgid "conflicting option string: %s"
msgid_plural "conflicting option strings: %s"
msgstr[0] "مقاطع خيارات متعارضة: %s"
msgstr[1] "سلسلة خيارات متعارضة: %s"
msgstr[2] "مقاطع خيارات متعارضة: %s"
msgstr[3] "مقاطع خيارات متعارضة: %s"
msgstr[4] "مقاطع خيارات متعارضة: %s"
msgstr[5] "مقاطع خيارات متعارضة: %s"

#: ../duplicity/argparse311.py:1595
msgid "mutually exclusive arguments must be optional"
msgstr "يجب أن تكون الحجج التي تستبعد بعضها البعض اختيارية"

#: ../duplicity/argparse311.py:1670
msgid "positional arguments"
msgstr "الحجج الموقعية"

#. Used in usage help. (Should be consistent with the "Options:"
#. header.) Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/argparse311.py:1671 ../duplicity/cli_data.py:364
#: ../duplicity/cli_data.py:529 ../duplicity/cli_data.py:583
#: ../duplicity/cli_data.py:718 ../duplicity/cli_data.py:918
msgid "options"
msgstr "خيارات"

#: ../duplicity/argparse311.py:1689
msgid "show this help message and exit"
msgstr "إظهار رسالة المساعدة هذه والخروج"

#: ../duplicity/argparse311.py:1721
msgid "cannot have multiple subparser arguments"
msgstr "لا يمكن أن يكون هناك حجج فرعية متعددة"

#: ../duplicity/argparse311.py:1769 ../duplicity/argparse311.py:2264
#, python-format
msgid "unrecognized arguments: %s"
msgstr "حجج غير معترف بها: %s"

#: ../duplicity/argparse311.py:1868
#, python-format
msgid "not allowed with argument %s"
msgstr "غير مسموح بالحجة %s"

#: ../duplicity/argparse311.py:1912 ../duplicity/argparse311.py:1926
#, python-format
msgid "ignored explicit argument %r"
msgstr ""

#: ../duplicity/argparse311.py:2030
#, python-format
msgid "the following arguments are required: %s"
msgstr "الحجج التالية مطلوبة: %s"

#: ../duplicity/argparse311.py:2042
#, python-format
msgid "one of the arguments %s is required"
msgstr "واحدة من الحجج %s مطلوبة"

#: ../duplicity/argparse311.py:2083
msgid "expected one argument"
msgstr "توقع حجة واحدة"

#: ../duplicity/argparse311.py:2084
msgid "expected at most one argument"
msgstr "المتوقع في حجة واحدة على الأكثر"

#: ../duplicity/argparse311.py:2085
msgid "expected at least one argument"
msgstr "توقع حجة واحدة على الأقل"

#: ../duplicity/argparse311.py:2089
#, python-format
msgid "expected %s argument"
msgid_plural "expected %s arguments"
msgstr[0] "توقّعت حجج %s"
msgstr[1] "سيارة %s المتوقعة"
msgstr[2] "توقّعت حجج %s"
msgstr[3] "توقّعت حجج %s"
msgstr[4] "توقّعت حجج %s"
msgstr[5] "توقّعت حجج %s"

#: ../duplicity/argparse311.py:2143
#, python-format
msgid "ambiguous option: %(option)s could match %(matches)s"
msgstr "خيار غامض: %(option)s يمكن أن يتطابق مع %(matches)s"

#: ../duplicity/argparse311.py:2207
#, python-format
msgid "unexpected option string: %s"
msgstr "سلسلة خيارات غير متوقعة: %s"

#: ../duplicity/argparse311.py:2402
#, python-format
msgid "%r is not callable"
msgstr ""

#: ../duplicity/argparse311.py:2419
#, python-format
msgid "invalid %(type)s value: %(value)r"
msgstr "قيمة %(type)s غير صالحة: %(value)r"

#: ../duplicity/argparse311.py:2429
#, python-format
msgid "invalid choice: %(value)r (choose from %(choices)s)"
msgstr "اختيار غير صالح: %(value)r (اختر من %(choices)s)"

#: ../duplicity/argparse311.py:2505
#, python-format
msgid "%(prog)s: error: %(message)s\n"
msgstr ""

#: ../duplicity/backend.py:95
#, python-brace-format
msgid "Import of {imp} Failed"
msgstr ""

#: ../duplicity/backend.py:200
#, python-format
msgid "Could not initialize backend: %s"
msgstr "تعذر تهيئة الخلفية: %s"

#: ../duplicity/backend.py:391
#, python-format
msgid "Backtrace of previous error: %s"
msgstr "خلفية الخطأ السابق: %s"

#: ../duplicity/backend.py:412
#, python-format
msgid "Giving up after %s attempts. %s: %s. (for trace back: set log level DEBUG)"
msgstr ""

#: ../duplicity/backend.py:421
#, python-format
msgid "Giving up after %s attempts. %s: %s"
msgstr "الاستسلام بعد محاولات %s . %s: %s"

#: ../duplicity/backend.py:427
#, python-format
msgid "Attempt of %s Nr. %s failed. %s: %s"
msgstr "فشلت محاولة %s Nr. %s . %s: %s"

#: ../duplicity/backend.py:526
#, python-format
msgid "Reading results of '%s'"
msgstr "نتائج القراءة من '%s'"

#: ../duplicity/backend.py:553
#, python-format
msgid "Writing %s"
msgstr "كتابة %s"

#: ../duplicity/backend.py:608
#, python-format
msgid "File %s not found locally after get from backend"
msgstr "لم يتم العثور على الملف %s محليا بعد الحصول من الخلفية"

#: ../duplicity/backend.py:661
#, python-format
msgid "%s Remote filesize %d for %s does not match local size %d"
msgstr ""

#: ../duplicity/backends/_testbackend.py:219
#, python-format
msgid "Unexpected exception while validate %s."
msgstr ""

#: ../duplicity/backends/giobackend.py:108
#, python-format
msgid "Connection failed, please check your password: %s"
msgstr "فشل الاتصال، الرجاء التحقق من كلمة المرور الخاصة بك: %s"

#: ../duplicity/backends/multibackend.py:100
#, python-format
msgid "MultiBackend: Could not parse query string %s: %s "
msgstr "متعدد الخلفية: لا يمكن تحليل سلسلة الاستعلام %s: %s "

#: ../duplicity/backends/multibackend.py:110
#, python-format
msgid "MultiBackend: Invalid query string %s: more than one value for %s"
msgstr "النسخ الخلفية: سلسلة الاستعلام غير صالحة %s: أكثر من قيمة واحدة ل %s"

#: ../duplicity/backends/multibackend.py:116
#, python-format
msgid "MultiBackend: Invalid query string %s: unknown parameter %s"
msgstr "النسخ الخلفية: سلسلة استعلام غير صالحة %s: معلمة غير معروفة %s"

#: ../duplicity/backends/multibackend.py:167
#: ../duplicity/backends/multibackend.py:174
#, python-format
msgid "MultiBackend: illegal value for %s: %s"
msgstr "متعدد الخلفية: قيمة غير قانونية ل %s: %s"

#: ../duplicity/backends/multibackend.py:186
#, python-format
msgid "MultiBackend: Url %s"
msgstr "متعدد الخلفات: رابط %s"

#: ../duplicity/backends/multibackend.py:189
#, python-format
msgid "MultiBackend: Could not load config file %s: %s "
msgstr "متعدد الخلفية: تعذر تحميل ملف الإعدادات %s: %s "

#: ../duplicity/backends/multibackend.py:196
#, python-format
msgid "MultiBackend: use store %s"
msgstr "متعدد الخلفية: استخدام متجر %s"

#: ../duplicity/backends/multibackend.py:200
#, python-format
msgid "MultiBackend: set env %s = %s"
msgstr "متعدد الخلفية: تعيين env %s = %s"

#: ../duplicity/backends/multibackend.py:214
#, python-format
msgid "Multibackend: register affinity for prefix %s"
msgstr "تعدد الخلفية: تسجيل صلة البادئة %s"

#: ../duplicity/backends/multibackend.py:257
#, python-format
msgid "MultiBackend: _put: write to store #%s (%s)"
msgstr "النسخ الخلفية: _put: الكتابة إلى مخزن #%s (%s)"

#: ../duplicity/backends/multibackend.py:272
#, python-format
msgid "MultiBackend: failed to write to store #%s (%s), try #%s, Exception: %s"
msgstr "النسخ الخلفية: فشل في الكتابة إلى مخزن #%s (%s)، حاول #%s، استثناء: %s"

#: ../duplicity/backends/multibackend.py:286
#, python-format
msgid "MultiBackend: failed to write %s. Aborting process."
msgstr "متعدد الخلفية: فشل في كتابة %s. إلغاء العملية."

#: ../duplicity/backends/multibackend.py:294
#, python-format
msgid "MultiBackend: failed to write %s. Tried all backing stores and none succeeded"
msgstr "متعدد الخلفية: فشل في كتابة %s. حاول جميع متاجر النسخ الاحتياطي ولم ينجح أي منها"

#: ../duplicity/backends/multibackend.py:315
#, python-format
msgid "MultiBackend: failed to get %s to %s from %s"
msgstr "متعدد الخلفات: فشل في الحصول على %s إلى %s من %s"

#: ../duplicity/backends/multibackend.py:320
#, python-format
msgid "MultiBackend: failed to get %s. Tried all backing stores and none succeeded"
msgstr "متعدد الخلفية: فشل في الحصول على %s. حاول جميع متاجر الدعم ولم ينجح أي منها"

#: ../duplicity/backends/multibackend.py:330
#, python-format
msgid "MultiBackend: %s: %d files"
msgstr "تعدد النسخ الخلفية: %s: %d ملفات"

#: ../duplicity/backends/multibackend.py:334
msgid "Exception during list of {s.backend.parsed_url.strip_auth()}: "
msgstr "استثناء أثناء قائمة {s.backend.parsed_url.strip_auth()}: "

#: ../duplicity/backends/multibackend.py:342
#, python-format
msgid "MultiBackend: combined list: %s"
msgstr "النسخ الخلفية: القائمة المدمجة: %s"

#: ../duplicity/backends/multibackend.py:371
#: ../duplicity/backends/multibackend.py:398
#, python-format
msgid "MultiBackend: failed to delete %s. Tried all backing stores and none succeeded"
msgstr "متعدد الخلفية: فشل في حذف %s. حاول جميع متاجر النسخ الاحتياطي ولم ينجح أي منها"

#: ../duplicity/backends/onedrivebackend.py:88
#, python-format
msgid "Attempt of initialize_oauth2_session Nr. %s failed. %s: %s"
msgstr "فشلت محاولة تهيئة_oauth2_الجلسة رقم %s . %s: %s"

#: ../duplicity/backends/pydrivebackend.py:203
#, python-format
msgid "PyDrive backend: multiple files called '%s'."
msgstr "خلفية PyDrive: ملفات متعددة تسمى '%s'."

#: ../duplicity/backends/webdavbackend.py:67
msgid "Missing socket or ssl python modules."
msgstr "مقبس أو وحدات sl python مفقودة."

#: ../duplicity/backends/webdavbackend.py:87
#, python-format
msgid "Cacert database file '%s' is not readable."
msgstr "ملف قاعدة البيانات '%s' غير قابل للقراءة."

#: ../duplicity/backends/webdavbackend.py:152
#, python-format
msgid "Using WebDAV host %s port %s"
msgstr "استخدام مضيف WebDAV %s منفذ %s"

#: ../duplicity/backends/webdavbackend.py:153
#, python-format
msgid "Using WebDAV directory %s"
msgstr "Using WebDAV directory %s"

#: ../duplicity/backends/webdavbackend.py:183
#, python-format
msgid "WebDAV create connection on '%s'"
msgstr "إنشاء اتصال WebDAV على '%s'"

#: ../duplicity/backends/webdavbackend.py:191
#, python-format
msgid "WebDAV Unknown URI scheme: %s"
msgstr "مخطط URI غير معروف WebDAV: %s"

#: ../duplicity/backends/webdavbackend.py:210
#, python-format
msgid "WebDAV %s %s request with headers: %s "
msgstr "WebDAV %s %s طلب مع العناوين الرئيسية: %s "

#: ../duplicity/backends/webdavbackend.py:211
#: ../duplicity/backends/webdavbackend.py:234
#, python-format
msgid "WebDAV data length: %s "
msgstr "طول بيانات WebDAV: %s "

#: ../duplicity/backends/webdavbackend.py:214
#, python-format
msgid "WebDAV response status %s with reason '%s'."
msgstr "حالة استجابة WebDAV %s بسبب '%s'."

#: ../duplicity/backends/webdavbackend.py:220
#, python-format
msgid "WebDAV redirect to: %s "
msgstr "إعادة توجيه WebDAV إلى: %s "

#: ../duplicity/backends/webdavbackend.py:222
msgid "WebDAV redirected 10 times. Giving up."
msgstr "تم إعادة توجيه WebDAV 10 مرات."

#: ../duplicity/backends/webdavbackend.py:227
msgid "WebDAV missing location header in redirect response."
msgstr "عنوان موقع WebDAV مفقود في إعادة توجيه الاستجابة."

#: ../duplicity/backends/webdavbackend.py:232
msgid "WebDAV retry request with authentification headers."
msgstr "إعادة محاولة طلب WebDAV مع عناوين المصادقة."

#: ../duplicity/backends/webdavbackend.py:233
#, python-format
msgid "WebDAV %s %s request2 with headers: %s "
msgstr "WebDAV %s %s طلب 2 مع العناوين الرئيسية: %s "

#: ../duplicity/backends/webdavbackend.py:237
#, python-format
msgid "WebDAV response2 status %s with reason '%s'."
msgstr "استجابات WebDAV الحالة %s مع سبب '%s'."

#: ../duplicity/backends/webdavbackend.py:256
msgid "python-kerberos needed to use kerberos                           authorization, falling back to basic auth."
msgstr "بايثون - كيروبيروس تحتاج إلى استخدام ترخيص الكيروبيروس، العودة إلى المصادقة الأساسية."

#: ../duplicity/backends/webdavbackend.py:264
#, python-format
msgid "Kerberos authorization failed: %s.                          Falling back to basic auth."
msgstr "فشل تفويض Kerberos: %s. العودة إلى المصادقة العادية."

#: ../duplicity/backends/webdavbackend.py:365
#, python-format
msgid "Creating missing directory %s"
msgstr "إنشاء الدليل مفقود %s"

#: ../duplicity/backends/webdavbackend.py:369
#, python-format
msgid "WebDAV MKCOL %s failed: %s %s"
msgstr "فشل WebDAV MKCOL %s : %s %s"

#: ../duplicity/backends/webdavbackend.py:381
#, python-format
msgid "WebDAV path decoding and translation: %s -> %s"
msgstr "فك ترميز وترجمة WebDAV: %s -> %s"

#: ../duplicity/backends/webdavbackend.py:417
#, python-format
msgid "WebDAV GET Bad status code %s reason %s."
msgstr "رمز حالة WebDAV سيء بسبب %s %s."

#: ../duplicity/backends/webdavbackend.py:438
#, python-format
msgid "WebDAV PUT Bad status code %s reason %s."
msgstr "رمز حالة PUT سيء بسبب %s %s."

#: ../duplicity/backends/webdavbackend.py:457
#, python-format
msgid "WebDAV DEL Bad status code %s reason %s."
msgstr "WebDAV DEL Bad status code %s reason %s."

#. Used in usage help to represent a Unix-style path name. Example:
#. --archive-dir <path>
#: ../duplicity/cli_data.py:103 ../duplicity/cli_data.py:173
#: ../duplicity/cli_data.py:207 ../duplicity/cli_data.py:255
#: ../duplicity/cli_data.py:358 ../duplicity/cli_data.py:381
#: ../duplicity/cli_data.py:552 ../duplicity/cli_data.py:744
#: ../duplicity/cli_data.py:927
msgid "path"
msgstr "المسار"

#: ../duplicity/cli_data.py:116
msgid "Hot|Cool|Archive"
msgstr "الأرشيف الساخن"

#. Used in usage help to represent a desired number of
#. something. Example:
#. --num-retries <number>
#: ../duplicity/cli_data.py:121 ../duplicity/cli_data.py:127
#: ../duplicity/cli_data.py:135 ../duplicity/cli_data.py:163
#: ../duplicity/cli_data.py:179 ../duplicity/cli_data.py:451
#: ../duplicity/cli_data.py:471 ../duplicity/cli_data.py:518
#: ../duplicity/cli_data.py:540 ../duplicity/cli_data.py:546
#: ../duplicity/cli_data.py:564 ../duplicity/cli_data.py:635
#: ../duplicity/cli_data.py:643 ../duplicity/cli_data.py:695
#: ../duplicity/cli_data.py:781 ../duplicity/cli_data.py:914
msgid "number"
msgstr "رقم"

#. Used in usage help. Example:
#. --timeout <seconds>
#: ../duplicity/cli_data.py:147 ../duplicity/cli_data.py:751
#: ../duplicity/cli_data.py:940
msgid "seconds"
msgstr "ثواني"

#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#: ../duplicity/cli_data.py:195 ../duplicity/cli_data.py:201
#: ../duplicity/cli_data.py:375 ../duplicity/cli_data.py:702
#: ../duplicity/cli_data.py:906
msgid "gpg-key-id"
msgstr "معرف المفتاح-gpg"

#. Used in usage help to represent a "glob" style pattern for
#. matching one or more files, as described in the documentation.
#. Example:
#. --exclude <shell_pattern>
#: ../duplicity/cli_data.py:212 ../duplicity/cli_data.py:403
#: ../duplicity/cli_data.py:945
msgid "shell_pattern"
msgstr "قذيفة_نقش"

#. Used in usage help to represent the name of a file. Example:
#. --log-file <filename>
#: ../duplicity/cli_data.py:224 ../duplicity/cli_data.py:230
#: ../duplicity/cli_data.py:291 ../duplicity/cli_data.py:409
#: ../duplicity/cli_data.py:903
msgid "filename"
msgstr "اسم الملف"

#. Used in usage help to represent a time spec for a previous
#. point in time, as described in the documentation. Example:
#. duplicity remove-older-than time [options] target_url
#: ../duplicity/cli_data.py:236 ../duplicity/cli_data.py:577
#: ../duplicity/cli_data.py:971
msgid "time"
msgstr "الوقت"

#: ../duplicity/cli_data.py:249 ../duplicity/cli_data.py:415
msgid "regex"
msgstr "ريجكس"

#: ../duplicity/cli_data.py:352
msgid "interval"
msgstr "الفاصل"

#: ../duplicity/cli_data.py:398
msgid "imap_mailbox"
msgstr "imap_mailbox"

#: ../duplicity/cli_data.py:423
msgid "If set, an extra file with statistics in json format will be stored with each backup. It includes the statistics printed out on stdout by default. With 'collection-status' it returs these stats."
msgstr "في حالة التعيين، سيتم تخزين ملف إضافي مع الإحصاءات في تنسيق json مع كل نسخة احتياطية. وهو يتضمن الإحصاءات المطبوعة على الاستدانة بشكل افتراضي. مع 'مجموعة - حالة' فإنه يعيد هذه الإحصاءات."

#: ../duplicity/cli_data.py:431
msgid "file_descriptor"
msgstr "وصف_ملف_"

#: ../duplicity/cli_data.py:438
msgid "log_filename"
msgstr "اسم الملف_log_filenname"

#: ../duplicity/cli_data.py:477
msgid "backup name"
msgstr "اسم النسخة الاحتياطية"

#: ../duplicity/cli_data.py:594
msgid "s3_endpoint_url"
msgstr "s3_endpoint_url"

#: ../duplicity/cli_data.py:662
msgid "s3_kms_key_id"
msgstr "s3_kms_key_id"

#: ../duplicity/cli_data.py:668
msgid "s3_kms_grant"
msgstr "منح s3_كمر_"

#: ../duplicity/cli_data.py:674
msgid "s3_region_name"
msgstr "اسم_المنطقة_s3_"

#: ../duplicity/cli_data.py:680
msgid "policy"
msgstr "السياسة"

#. noun
#: ../duplicity/cli_data.py:685 ../duplicity/cli_data.py:690
#: ../duplicity/cli_data.py:892
msgid "command"
msgstr "أمر"

#: ../duplicity/cli_data.py:730
msgid "pem formatted bundle of certificate authorities"
msgstr "الحزمة المنسقة لسلطات الشهادة"

#: ../duplicity/cli_data.py:735
msgid "path to a folder with certificate authority files"
msgstr "مسار إلى مجلد يحتوي على ملفات مرخص بالشهادة"

#. abbreviation for "character" (noun)
#: ../duplicity/cli_data.py:757 ../duplicity/cli_data.py:890
msgid "char"
msgstr "حرف"

#: ../duplicity/cli_data.py:770
msgid "verb"
msgstr "فعل"

#: ../duplicity/cli_data.py:788
msgid "extra headers for Webdav, like 'Cookie,name=value'"
msgstr "رؤوس إضافية لـ Webdav، مثل 'Cookie,name=value'"

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other_host[:port]//absolute_path
#: ../duplicity/cli_data.py:881
msgid "absolute_path"
msgstr "مسار_مطلقة"

#. Used in usage help. Example:
#. tahoe://alias/some_dir
#: ../duplicity/cli_data.py:884
msgid "alias"
msgstr "alias"

#. Used in help to represent a "bucket name" for Amazon Web
#. Services' Simple Storage Service (S3). Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/cli_data.py:888
msgid "bucket_name"
msgstr "إسم_دلو"

#. Used in usage help to represent the name of a container in
#. Amazon Web Services' Cloudfront. Example:
#. cf+http://container_name
#: ../duplicity/cli_data.py:896
msgid "container_name"
msgstr "اسم الحاوية_"

#. noun
#: ../duplicity/cli_data.py:898
msgid "count"
msgstr "العد"

#. Used in usage help to represent the name of a file directory
#: ../duplicity/cli_data.py:900
msgid "directory"
msgstr "دليل"

#. Used in usage help, e.g. to represent the name of a code
#. module. Example:
#. rsync://user[:password]@other.host[:port]::/module/some_dir
#: ../duplicity/cli_data.py:910
msgid "module"
msgstr "الوحدة"

#. Used in usage help to represent an internet hostname. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:921
msgid "other.host"
msgstr "آخر.المضيف"

#. Used in usage help. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:924
msgid "password"
msgstr "كلمة المرور"

#. Used in usage help to represent a TCP port number. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:930
msgid "port"
msgstr "منفذ"

#. Used in usage help. This represents a string to be used as a
#. prefix to names for backup files created by Duplicity. Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/cli_data.py:934
msgid "prefix"
msgstr "بادئة"

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other.host[:port]/relative_path
#: ../duplicity/cli_data.py:937
msgid "relative_path"
msgstr "مسار_نسبي"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. Example:
#. file:///some_dir
#: ../duplicity/cli_data.py:949
msgid "some_dir"
msgstr "عدد_ديري"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory where files will be
#. coming FROM. Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/cli_data.py:954
msgid "source_path"
msgstr "المصدر_المسار"

#. Used in usage help to represent a URL files will be coming
#. FROM. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/cli_data.py:958
msgid "source_url"
msgstr "مصدر_رابط"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. where files will be
#. going TO. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/cli_data.py:963
msgid "target_dir"
msgstr "target_dir"

#. Used in usage help to represent a URL files will be going TO.
#. Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/cli_data.py:967
msgid "target_url"
msgstr "target_url"

#. Used in usage help to represent a user name (i.e. login).
#. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:975
msgid "user"
msgstr "مستخدم"

#. account id for b2. Example: b2://account_id@bucket/
#: ../duplicity/cli_data.py:977
msgid "account_id"
msgstr "معرف الحساب_"

#. application_key for b2.
#. Example: b2://account_id:application_key@bucket/
#: ../duplicity/cli_data.py:980
msgid "application_key"
msgstr "تطبيق_مفتاح"

#. remote name for rclone.
#. Example: rclone://remote:/some_dir
#: ../duplicity/cli_data.py:983 ../duplicity/dup_collections.py:565
msgid "remote"
msgstr "عن بعد"

#: ../duplicity/cli_data.py:987
msgid "Backends and their URL formats:"
msgstr "النسخ الخلفية وصيغ عنوان URL الخاصة بها:"

#: ../duplicity/cli_main.py:151
msgid "No valid action found. Will imply 'backup' because a path source was given and target is a url location."
msgstr "لم يتم العثور على إجراء صحيح. سوف يعني ضمناً \"النسخ الاحتياطي\" لأن مصدر المسار تم إعطاؤه والهدف هو موقع الرابط."

#: ../duplicity/cli_main.py:159
msgid "No valid action found. Will imply 'restore' because url source was given and target is a local path."
msgstr "لم يتم العثور على إجراء صحيح. سوف يعني ضمناً 'الاستعادة' لأن مصدر url قد تم إعطاؤه والهدف هو المسار المحلي."

#: ../duplicity/cli_main.py:166
#, python-brace-format
msgid "Invalid '{remainder[0]}' action and cannot be implied from the "
msgstr "إجراء '{تبقي[0]}' غير صالح ولا يمكن أن يكون ضمنيا من "

#: ../duplicity/cli_main.py:266
#, python-brace-format
msgid "GPG binary is {config.gpg_binary}, version {gpg_version}"
msgstr "GPG الثنائي هو {config.gpg_binary}، الإصدار {gpg_version}"

#: ../duplicity/cli_main.py:315
#, python-brace-format
msgid "Using archive dir: {config.archive_dir_path.uc_name}"
msgstr "استخدام أغلفة الأرشيف: {config.archive_dir_path.uc_name}"

#: ../duplicity/cli_main.py:316
#, python-brace-format
msgid "Using backup name: {config.backup_name}"
msgstr "استخدام اسم النسخ الاحتياطي: {config.backup_name}"

#: ../duplicity/cli_util.py:45
msgid "Enter 'duplicity --help' for help screen."
msgstr "أدخل 'الازدواجية --المساعدة' لشاشة المساعدة."

#: ../duplicity/cli_util.py:127
msgid "Running in 'ignore errors' mode due to --ignore-errors.\n"
"Please reconsider if this was not intended"
msgstr ""

#: ../duplicity/cli_util.py:170
msgid "'{val}' is not an int: {str(e)}"
msgstr "''{val}' ليس صحيح: {str(e)}"

#: ../duplicity/cli_util.py:184
#, python-brace-format
msgid "'{val} is not a single character."
msgstr "{val} ليس حرف واحد."

#: ../duplicity/cli_util.py:211
#, python-brace-format
msgid "Source should be pathname, not url.  Got '{val}' instead."
msgstr "يجب أن يكون المصدر اسم المسار، وليس عنوان url. احصل على '{val}' بدلاً من ذلك."

#: ../duplicity/cli_util.py:213
#, python-brace-format
msgid "Argument source_path '{val}' does not exist."
msgstr "Argument source_path '{val}' غير موجود."

#: ../duplicity/cli_util.py:219
#, python-brace-format
msgid "Source should be url, not directory.  Got '{val}' instead."
msgstr "يجب أن يكون المصدر عنوان url، وليس الدليل. حصلت على '{val}' بدلاً من ذلك."

#: ../duplicity/cli_util.py:225
#, python-brace-format
msgid "Target should be directory, not url.  Got '{val}' instead."
msgstr "يجب أن يكون الهدف الدليل، وليس العنوان. الحصول على '{val}' بدلاً من ذلك."

#: ../duplicity/cli_util.py:230
msgid "Unable to create target dir '{val}': {str(e)}"
msgstr "غير قادر على إنشاء هدف dir '{val}': {str(e)}"

#: ../duplicity/cli_util.py:236
#, python-brace-format
msgid "Target should be url, not directory.  Got '{val}' instead."
msgstr "يجب أن يكون الهدف عنوان urt، وليس الدليل. الحصول على '{val}' بدلاً من ذلك."

#: ../duplicity/cli_util.py:281
msgid "Verbosity must be one of: digit [0-9], character [ewnid],\n"
"or word ['error', 'warning', 'notice', 'info', 'debug'].\n"
"The default is 3 (Notice).  It is strongly recommended\n"
"that verbosity level is set at 2 (Warning) or higher."
msgstr ""

#: ../duplicity/cli_util.py:425
#, python-brace-format
msgid "Specified archive directory '{archive_dir_path.uc_name}' is not a directory"
msgstr "دليل الأرشيف المحدد '{archive_dir_path.uc_name}' ليس مجلد"

#: ../duplicity/cli_util.py:434
msgid "Encrypt key should be an 8, 16, or 40 character hex string, like 'AA0E73D2'.\n"
msgstr "يجب أن يكون مفتاح التشفير سلسلة هوس حروف 8 أو 16 أو 40 حرف، مثل 'A0E73D2'.\n"

#: ../duplicity/cli_util.py:454
msgid "Hidden dncrypt key should be an 8, 16, or 40 character hex string, like 'AA0E73D2'.\n"
msgstr "يجب أن يكون مفتاح التشفير المخفي 8 أو 16 أو 40 حرف، مثل 'A0E73D2'.\n"

#: ../duplicity/cli_util.py:468
msgid "Sign key should be an 8, 16, or 40 character hex string, like 'AA0E73D2'.\n"
msgstr "يجب أن يكون مفتاح التوقيع 8 أو 16 أو 40 حرف، مثل 'A0E73D2'.\n"

#: ../duplicity/diffdir.py:105 ../duplicity/diffdir.py:404
#, python-format
msgid "Error %s getting delta for %s"
msgstr "خطأ %s الحصول على دلتا ل %s"

#: ../duplicity/diffdir.py:119
#, python-format
msgid "Getting delta of %s and %s"
msgstr "الحصول على دلتا من %s و %s"

#: ../duplicity/diffdir.py:163
#, python-format
msgid "A %s"
msgstr "%s"

#: ../duplicity/diffdir.py:171
#, python-format
msgid "M %s"
msgstr ""

#: ../duplicity/diffdir.py:194
#, python-format
msgid "Comparing %s and %s"
msgstr "مقارنة %s و %s"

#: ../duplicity/diffdir.py:207
#, python-format
msgid "D %s"
msgstr "D %s"

#: ../duplicity/dup_collections.py:200 ../duplicity/dup_collections.py:209
#, python-format
msgid "BackupSet.delete: missing %s"
msgstr "النسخ الاحتياطي.حذف: مفقود %s"

#: ../duplicity/dup_collections.py:235
msgid "Fatal Error: No manifests found for most recent backup"
msgstr "خطأ مميت: لا توجد بيانات لآخر نسخة احتياطية"

#: ../duplicity/dup_collections.py:248
msgid "Fatal Error: Remote manifest does not match local one.  Either the remote backup set or the local archive directory has been corrupted."
msgstr "خطأ مميت: البيان عن بعد لا يتطابق مع البيان المحلي. إما أن مجموعة النسخ الاحتياطي عن بعد أو دليل الأرشيف المحلي قد تعطل."

#: ../duplicity/dup_collections.py:256
#, python-format
msgid "Error processing remote manifest (%s): %s"
msgstr "خطأ في معالجة البيان البعيد (%s: %s"

#: ../duplicity/dup_collections.py:265
msgid "Fatal Error: Neither remote nor local manifest is readable."
msgstr "خطأ مميت: لا يمكن قراءة البيان عن بعد أو عن طريق محلي."

#: ../duplicity/dup_collections.py:276
#, python-format
msgid "Processing local manifest %s (%s)"
msgstr "معالجة البيان المحلي %s (%s)"

#: ../duplicity/dup_collections.py:291
msgid "Error processing remote file ({os.fsdecode(remote_file)}): {util.uexc(message)}"
msgstr "خطأ في معالجة الملف البعيد ({os.fsdecode(remote_file)}): {util.uexc(message)}"

#: ../duplicity/dup_collections.py:293
msgid "Processing remote file {os.fsdecode(remote_file)} ({len(remote_file_buffer)})"
msgstr "معالجة الملف البعيد {os.fsdecode(remote_file)} ({len(remote_file_buffer)})"

#: ../duplicity/dup_collections.py:311
msgid "No Jsonstat file found, return empty."
msgstr ""

#: ../duplicity/dup_collections.py:404
msgid "Preferring Backupset over previous one!"
msgstr "تفضيل النسخ الاحتياطي على النسخة السابقة!"

#: ../duplicity/dup_collections.py:408
#, python-format
msgid "Ignoring incremental Backupset (start_time: %s; needed: %s)"
msgstr "تجاهل النسخ الاحتياطي التراكمي (start_time: %s; مطلوب: %s)"

#: ../duplicity/dup_collections.py:417
#, python-format
msgid "Added incremental Backupset (start_time: %s / end_time: %s)"
msgstr "تم إضافة نسخة احتياطية إضافية (start_time: %s / end_time: %s)"

#: ../duplicity/dup_collections.py:490
msgid "Chain start time: "
msgstr "وقت بداية السلسلة: "

#: ../duplicity/dup_collections.py:491
msgid "Chain end time: "
msgstr "وقت نهاية السلسلة: "

#: ../duplicity/dup_collections.py:492
#, python-format
msgid "Number of contained backup sets: %d"
msgstr "عدد مجموعات النسخ الاحتياطي المحتوى: %d"

#: ../duplicity/dup_collections.py:493
#, python-format
msgid "Total number of contained volumes: %d"
msgstr "العدد الإجمالي للمجلدات المحتوى: %d"

#: ../duplicity/dup_collections.py:494 ../duplicity/dup_collections.py:1301
msgid "Type of backup set:"
msgstr "نوع مجموعة النسخ الاحتياطي:"

#: ../duplicity/dup_collections.py:494 ../duplicity/dup_collections.py:1301
msgid "Time:"
msgstr "الوقت:"

#: ../duplicity/dup_collections.py:494
msgid "Num volumes:"
msgstr "المجلدات النوم:"

#: ../duplicity/dup_collections.py:499 ../duplicity/dup_collections.py:1308
msgid "Full"
msgstr "كامل"

#: ../duplicity/dup_collections.py:502 ../duplicity/dup_collections.py:1310
msgid "Incremental"
msgstr "زيادة"

#: ../duplicity/dup_collections.py:563
msgid "local"
msgstr "محلي"

#: ../duplicity/dup_collections.py:726
msgid "Collection Status"
msgstr "حالة المجموعة"

#: ../duplicity/dup_collections.py:728
#, python-format
msgid "Connecting with backend: %s"
msgstr "الاتصال مع الخلفية: %s"

#: ../duplicity/dup_collections.py:729
#, python-format
msgid "Archive dir: %s"
msgstr "قذيفة الأرشيف: %s"

#: ../duplicity/dup_collections.py:732
#, python-format
msgid "Found %d secondary backup chain(s)."
msgstr "تم العثور على %d سلسلة احتياطية ثانوية."

#: ../duplicity/dup_collections.py:734
#, python-format
msgid "Secondary chain %d of %d:"
msgstr "السلسلة الثانوية %d من %d:"

#: ../duplicity/dup_collections.py:739
msgid "Found primary backup chain with matching signature chain:"
msgstr "تم العثور على سلسلة النسخ الاحتياطي الأساسية مع سلسلة توقيع مطابقة:"

#: ../duplicity/dup_collections.py:742
msgid "No backup chains with active signatures found"
msgstr "لا توجد سلاسل احتياطية مع التوقيعات النشطة"

#: ../duplicity/dup_collections.py:745
#, python-format
msgid "Also found %d backup set(s) not part of any chain,"
msgstr "ايضا وجدت %d مجموعة (مجموعات) نسخ احتياطي ليست جزءا من أي سلسلة،"

#: ../duplicity/dup_collections.py:746
#, python-format
msgid "and %d incomplete backup set(s)."
msgstr "و %d نسخة احتياطية غير مكتملة."

#. "cleanup" is a hard-coded command, so do not translate it
#: ../duplicity/dup_collections.py:748
msgid "These may be deleted by running duplicity with the \"cleanup\" command."
msgstr "يمكن حذف هذه عن طريق تشغيل الازدواجية مع أمر \"التنظيف\"."

#: ../duplicity/dup_collections.py:750
msgid "No orphaned or incomplete backup sets found."
msgstr "لم يتم العثور على مجموعات احتياطية أيتام أو غير كاملة."

#: ../duplicity/dup_collections.py:766
#, python-format
msgid "%d file(s) exists on backend"
msgstr "يوجد %d ملف (ملفات) في الخلفية"

#: ../duplicity/dup_collections.py:770
#, python-format
msgid "%d file(s) exist in cache"
msgstr ""

#: ../duplicity/dup_collections.py:824
msgid "Warning, discarding last backup set, because of missing signature file."
msgstr "تحذير، تجاهل آخر مجموعة احتياطية بسبب فقدان ملف التوقيع."

#: ../duplicity/dup_collections.py:847
msgid "Warning, found the following local orphaned signature file(s):"
msgstr "تحذير، عثر على ملف (ملفات) التوقيع المحلي اليتامى التالي:"

#: ../duplicity/dup_collections.py:855
msgid "Warning, found the following remote orphaned signature file(s):"
msgstr "تحذير، عثر على ملف (ملفات) التوقيع اليتامى عن بعد:"

#: ../duplicity/dup_collections.py:863
msgid "Warning, found signatures but no corresponding backup files"
msgstr "تحذير، تم العثور على التوقيعات ولكن لا توجد ملفات النسخ الاحتياطي المقابلة"

#: ../duplicity/dup_collections.py:869
msgid "Warning, found incomplete backup sets, probably left from aborted session"
msgstr "تحذير ، تم العثور على مجموعات غير مكتملة للنسخ الاحتياطي، ربما تبقى من جلسة محلولة"

#: ../duplicity/dup_collections.py:875
msgid "Warning, found the following orphaned backup file(s):"
msgstr "تحذير، عثر على ملف (ملفات) النسخ الاحتياطي اليتامى التالية:"

#: ../duplicity/dup_collections.py:890
#, python-format
msgid "Extracting backup chains from list of files: %s"
msgstr "استخراج سلاسل النسخ الاحتياطي من قائمة الملفات: %s"

#: ../duplicity/dup_collections.py:901
#, python-format
msgid "File %s is part of known set"
msgstr "الملف %s هو جزء من مجموعة معروفة"

#: ../duplicity/dup_collections.py:904
#, python-format
msgid "File %s is not part of a known set; creating new set"
msgstr "الملف %s ليس جزءا من مجموعة معروفة؛ إنشاء مجموعة جديدة"

#: ../duplicity/dup_collections.py:909
#, python-format
msgid "Ignoring file (rejected by backup set) '%s'"
msgstr "تجاهل الملف (مرفوضة بواسطة مجموعة النسخ الاحتياطي) '%s'"

#: ../duplicity/dup_collections.py:925
#, python-format
msgid "Found backup chain %s"
msgstr "تم العثور على سلسلة النسخ الاحتياطي %s"

#: ../duplicity/dup_collections.py:930
#, python-format
msgid "Added set %s to pre-existing chain %s"
msgstr "تم إضافة مجموعة %s إلى سلسلة موجودة مسبقاً %s"

#: ../duplicity/dup_collections.py:933
#, python-format
msgid "Found orphaned set %s"
msgstr "تم العثور على مجموعة أيتام %s"

#: ../duplicity/dup_collections.py:1081
#, python-format
msgid "No signature chain for the requested time. Using oldest available chain, starting at time %s."
msgstr "لا توجد سلسلة توقيع للوقت المطلوب. استخدام أقدم سلسلة متاحة، بدءاً من الوقت %s."

#: ../duplicity/dup_collections.py:1299
#, python-format
msgid "File: %s"
msgstr "الملف: %s"

#: ../duplicity/dup_collections.py:1300
#, python-format
msgid "Total number of backup: %d"
msgstr "إجمالي عدد النسخ الاحتياطي: %d"

#: ../duplicity/dup_collections.py:1301 ../duplicity/dup_collections.py:1337
msgid "Type of file change:"
msgstr "نوع تغيير الملف:"

#: ../duplicity/dup_collections.py:1335
#, python-format
msgid " Backup set time: %s"
msgstr " وقت إعداد النسخ الاحتياطي: %s"

#: ../duplicity/dup_collections.py:1336
#, python-format
msgid "Total number of changes: %d"
msgstr "مجموع عدد التغييرات: %d"

#: ../duplicity/dup_collections.py:1337
msgid "File:"
msgstr "الملف:"

#: ../duplicity/dup_main.py:109
msgid "Reuse configured PASSPHRASE as SIGN_PASSPHRASE"
msgstr "إعادة استخدام التهيئة كـ SIGN_PASSPHRASE"

#: ../duplicity/dup_main.py:120
msgid "Reuse configured SIGN_PASSPHRASE as PASSPHRASE"
msgstr "إعادة استخدام SIGN_PASSPHRASE كباسسراسي"

#: ../duplicity/dup_main.py:155
msgid "PASSPHRASE variable not set, asking user."
msgstr "لم يتم تعيين متغير PASSPHRASE ، يسأل المستخدم."

#: ../duplicity/dup_main.py:180
msgid "Retype passphrase for signing key to confirm: "
msgstr "إعادة كتابة عبارة المرور لمفتاح التوقيع للتأكيد: "

#: ../duplicity/dup_main.py:182
msgid "Retype passphrase for decryption to confirm: "
msgstr "أعد كتابة عبارة المرور لفك التشفير لتأكيد: "

#: ../duplicity/dup_main.py:186
msgid "First and second passphrases do not match!  Please try again."
msgstr "عبارتا المرور الأولى والثانية غير متطابقتين! الرجاء المحاولة مرة أخرى."

#: ../duplicity/dup_main.py:197
msgid "Cannot use empty passphrase with symmetric encryption!  Please try again."
msgstr "لا يمكن استخدام كلمة مرور فارغة مع تشفير متناظر! الرجاء المحاولة مرة أخرى."

#: ../duplicity/dup_main.py:257
#, python-format
msgid "File %s complete in backup set.\n"
"Continuing restart on file %s."
msgstr "الملف %s اكتمل في مجموعة النسخ الاحتياطي.\n"
"إعادة التشغيل على الملف %s."

#: ../duplicity/dup_main.py:267
#, python-format
msgid "File %s missing in backup set.\n"
"Continuing restart on file %s."
msgstr "الملف %s مفقود في مجموعة النسخ الاحتياطي.\n"
"مواصلة إعادة التشغيل على الملف %s."

#: ../duplicity/dup_main.py:322
#, python-format
msgid "File %s was corrupted during upload."
msgstr "الملف %s تالف أثناء التحميل."

#: ../duplicity/dup_main.py:352
msgid "Restarting backup, but current encryption settings do not match original settings"
msgstr "إعادة تشغيل النسخ الاحتياطي، ولكن إعدادات التشفير الحالية لا تتطابق مع الإعدادات الأصلية"

#: ../duplicity/dup_main.py:375 ../duplicity/dup_main.py:511
#, python-format
msgid "Processed volume %d"
msgstr "تم معالجة مستوى الصوت %d"

#: ../duplicity/dup_main.py:423
msgid "Skipping encryption validation due to glacier/deep storage"
msgstr "تخطي التحقق من التشفير بسبب التخزين الجليدي/العميق"

#: ../duplicity/dup_main.py:427
#, python-format
msgid "Restarting after volume %s, file %s, block %s"
msgstr "إعادة التشغيل بعد المجلد %s، الملف %s، حظر %s"

#: ../duplicity/dup_main.py:479
msgid "Skipped volume upload, as effectivly nothing has changed"
msgstr "تخطي رفع مستوى الصوت، حيث لم يتغير شيء"

#: ../duplicity/dup_main.py:702
msgid "Fatal Error: Unable to start incremental backup.  Old signatures not found and incremental specified"
msgstr "خطأ قاتل: غير قادر على بدء النسخ الاحتياطي الإضافي. التوقيعات القديمة غير موجودة والمحددة بشكل تدريجي"

#: ../duplicity/dup_main.py:708
msgid "No signatures found, switching to full backup."
msgstr "لم يتم العثور على توقيعات، التبديل إلى النسخة الاحتياطية الكاملة."

#: ../duplicity/dup_main.py:722
msgid "Backup Statistics"
msgstr "إحصائيات النسخ الاحتياطي"

#: ../duplicity/dup_main.py:862
#, python-format
msgid "%s not found in archive - no files restored."
msgstr "لم يتم العثور على %s في الأرشيف - لم يتم استعادة أي ملفات."

#: ../duplicity/dup_main.py:866
msgid "No files found in archive - nothing restored."
msgstr "لم يتم العثور على ملفات في الأرشيف - لم يتم استعادة أي شيء."

#: ../duplicity/dup_main.py:904
#, python-format
msgid "Processed volume %d of %d"
msgstr "تم معالجة المجلد %d من %d"

#: ../duplicity/dup_main.py:947
#, python-format
msgid "Invalid data - %s hash mismatch for file:"
msgstr "بيانات غير صالحة - %s عدم تطابق التجزئة للملف:"

#: ../duplicity/dup_main.py:949
#, python-format
msgid "Calculated hash: %s"
msgstr "التجزئة المحسوبة: %s"

#: ../duplicity/dup_main.py:950
#, python-format
msgid "Manifest hash: %s"
msgstr "تجزئة البيان: %s"

#: ../duplicity/dup_main.py:957 ../duplicity/util.py:110
#, python-format
msgid "IGNORED_ERROR: Warning: ignoring error as requested: %s: %s"
msgstr "IGNORED_خطأ: تحذير: تجاهل الخطأ كما هو مطلوب: %s: %s"

#: ../duplicity/dup_main.py:1004
#, python-format
msgid "Volume was signed by key %s, not %s"
msgstr "تم توقيع المجلد بواسطة المفتاح %sوليس %s"

#: ../duplicity/dup_main.py:1036
#, python-format
msgid "Verify complete: %s, %s."
msgstr "تم التحقق: %s، %s."

#: ../duplicity/dup_main.py:1037
#, python-format
msgid "%d file(s) compared"
msgstr "%d ملف (ملفات) مقارنة"

#: ../duplicity/dup_main.py:1037
#, python-format
msgid "%d difference(s) found"
msgstr ""

#: ../duplicity/dup_main.py:1056
msgid "No extraneous files found, nothing deleted in cleanup."
msgstr "لم يتم العثور على ملفات خارجية، لم يتم حذف أي شيء في التنظيف."

#: ../duplicity/dup_main.py:1071
msgid "Found the following file(s) to delete:"
msgstr "العثور على الملف (الملفات) التالي للحذف:"

#: ../duplicity/dup_main.py:1075
msgid "Run duplicity again with the --force option to actually delete."
msgstr "تشغيل الازدواجية مرة أخرى مع - خيار القوة للحذف بالفعل."

#: ../duplicity/dup_main.py:1121
msgid "There are backup set(s) at time(s):"
msgstr "هناك مجموعة (مجموعات) النسخ الاحتياطي في الوقت (الوقت):"

#: ../duplicity/dup_main.py:1123
msgid "Which can't be deleted because newer sets depend on them."
msgstr "ما لا يمكن حذفه لأن المجموعات الأحدث تعتمد عليها."

#: ../duplicity/dup_main.py:1130
msgid "Current active backup chain is older than specified time.  However, it will not be deleted.  To remove all your backups, manually purge the repository."
msgstr "سلسلة النسخ الاحتياطي النشطة الحالية قديمة عن الوقت المحدد. ومع ذلك، لن يتم حذفها. لإزالة جميع النسخ الاحتياطية الخاصة بك، قم بإزالة المستودع يدويا."

#: ../duplicity/dup_main.py:1148
msgid "No old backup sets found, nothing deleted."
msgstr "لم يتم العثور على مجموعات النسخ الاحتياطي القديمة، لم يتم حذف أي شيء."

#: ../duplicity/dup_main.py:1151
msgid "Deleting backup chain(s) at time:"
msgstr "حذف سلسلة (سلاسل) النسخ الاحتياطي في الوقت المناسب:"

#: ../duplicity/dup_main.py:1160
#, python-format
msgid "Deleting any incremental signature chain rooted at %s"
msgstr "حذف أي سلسلة توقيع تصاعدية متجذرة في %s"

#: ../duplicity/dup_main.py:1162
#, python-format
msgid "Deleting any incremental backup chain rooted at %s"
msgstr "حذف أي سلسلة نسخ احتياطية إضافية متجذرة في %s"

#: ../duplicity/dup_main.py:1165
#, python-format
msgid "Deleting complete signature chain %s"
msgstr "حذف سلسلة توقيع كاملة %s"

#: ../duplicity/dup_main.py:1167
#, python-format
msgid "Deleting complete backup chain %s"
msgstr "حذف سلسلة النسخ الاحتياطي الكاملة %s"

#: ../duplicity/dup_main.py:1174
msgid "Found old backup chain(s) at the following time:"
msgstr "العثور على سلسلة النسخ الاحتياطي القديمة في الوقت التالي:"

#: ../duplicity/dup_main.py:1178
msgid "Rerun command with --force option to actually delete."
msgstr "أمر Rerun مع --إجبار الخيار للحذف بالفعل."

#: ../duplicity/dup_main.py:1281
#, python-format
msgid "Deleting local %s (not authoritative at backend)."
msgstr "حذف %s المحلية (غير مرخص في الخلفية)."

#: ../duplicity/dup_main.py:1285
#, python-format
msgid "Unable to delete %s: %s"
msgstr "غير قادر على حذف %s: %s"

#: ../duplicity/dup_main.py:1319 ../duplicity/dup_temp.py:296
#, python-format
msgid "Failed to read %s: %s"
msgstr "فشل في قراءة %s: %s"

#: ../duplicity/dup_main.py:1331
#, python-format
msgid "Copying %s to local cache."
msgstr "نسخ %s إلى ذاكرة التخزين المؤقت المحلية."

#: ../duplicity/dup_main.py:1379
msgid "Local and Remote metadata are synchronized, no sync needed."
msgstr "البيانات الوصفية المحلية والبعيدة متزامنة، لا حاجة للمزامنة."

#: ../duplicity/dup_main.py:1384
msgid "Synchronizing remote metadata to local cache..."
msgstr "مزامنة بيانات التعريف عن بعد إلى ذاكرة التخزين المؤقت المحلية..."

#: ../duplicity/dup_main.py:1398
msgid "Sync would copy the following from remote to local:"
msgstr "سيتم نسخ المزامنة التالية من بعد إلى محلي:"

#: ../duplicity/dup_main.py:1404
msgid "Sync would remove the following spurious local files:"
msgstr "المزامنة ستزيل الملفات المحلية الزائفة التالية:"

#: ../duplicity/dup_main.py:1450
msgid "Unable to get free space on temp."
msgstr "غير قادر على الحصول على مساحة مجانية في درجة الحرارة."

#: ../duplicity/dup_main.py:1457
msgid "Temp space has {freespace:,} available, backup needs approx {needspace:,}."
msgstr ""

#: ../duplicity/dup_main.py:1461
msgid "Temp has {freespace:,} available, backup will use approx {needspace:,}."
msgstr ""

#: ../duplicity/dup_main.py:1468
msgid "Unable to get max open files."
msgstr "غير قادر على الحصول على الحد الأقصى للملفات المفتوحة."

#: ../duplicity/dup_main.py:1473
#, python-brace-format
msgid "Max open files of {maxopen} is too low, should be >= 1024.\n"
msgstr ""

#: ../duplicity/dup_main.py:1528
msgid "RESTART: The first volume failed to upload before termination.\n"
"         Restart is impossible...starting backup from beginning."
msgstr ""

#: ../duplicity/dup_main.py:1538
#, python-format
msgid "RESTART: Volumes %d to %d failed to upload before termination.\n"
"         Restarting backup at volume %d."
msgstr "RESTART: فشل تحميل المجلدات %d إلى %d قبل الإنهاء.\n"
"         إعادة تشغيل النسخ الاحتياطي في المجلد %d."

#: ../duplicity/dup_main.py:1549
#, python-format
msgid "RESTART: Impossible backup state: manifest has %d vols, remote has %d vols.\n"
"         Restart is impossible ... duplicity will clean off the last partial\n"
"         backup then restart the backup from the beginning."
msgstr "RESTART: حالة النسخ الاحتياطي المستحيلة: البيان لديه %d مجلدات، عن بعد %d مجلدات.\n"
"         إعادة التشغيل مستحيلة. . سيتم مسح الازدواجية من النسخة الاحتياطية الجزئية\n"
"         بعد إعادة تشغيل النسخة الاحتياطية من البداية."

#: ../duplicity/dup_main.py:1575
msgid "                PYTHONOPTIMIZE in the environment causes duplicity to fail to\n"
"                recognize its own backups.  Please remove PYTHONOPTIMIZE from\n"
"                the environment and rerun the backup.\n\n"
"                See https://bugs.launchpad.net/duplicity/+bug/931175\n"
"                "
msgstr ""

#: ../duplicity/dup_main.py:1655
#, python-brace-format
msgid "Last {action} backup left a partial set, restarting."
msgstr "آخر نسخة احتياطية {action} تركت مجموعة جزئية، إعادة التشغيل."

#: ../duplicity/dup_main.py:1659
#, python-brace-format
msgid "Cleaning up previous partial {action} backup set, restarting."
msgstr "تنظيف مجموعة النسخ الاحتياطي الجزئية {action} السابقة، إعادة التشغيل."

#: ../duplicity/dup_main.py:1673
msgid "Last full backup date: none"
msgstr "تاريخ النسخ الاحتياطي الأخير: لا يوجد"

#: ../duplicity/dup_main.py:1680
msgid "Last full backup is too old, forcing full backup"
msgstr "آخر نسخة احتياطية كاملة قديمة جداً، مما يجبر على النسخ الاحتياطي الكامل"

#: ../duplicity/dup_main.py:1736
msgid "When using symmetric encryption, the signing passphrase must equal the encryption passphrase."
msgstr "عند استخدام التشفير المتماثل، يجب أن تعادل عبارة مرور كلمة مرور التشفير."

#: ../duplicity/dup_time.py:54
#, python-format
msgid "Bad interval string \"%s\"\n\n"
"Intervals are specified like 2Y (2 years) or 2h30m (2.5 hours).  The\n"
"allowed special characters are s, m, h, D, W, M, and Y.  See the man\n"
"page for more information."
msgstr "سلسلة الفاصل الزمني السيئة \"%s\"\n\n"
"تحدد مثل 2Y (2 سنة) أو 2h30m (2.5 ساعة).\n"
"الأحرف الخاصة المسموح بها هي , m, h, D, W, M, Y. راجع صفحة الرجل\n"
"لمزيد من المعلومات."

#: ../duplicity/dup_time.py:63
#, python-format
msgid "Bad time string \"%s\"\n\n"
"The acceptible time strings are intervals (like \"3D64s\"), w3-datetime\n"
"strings, like \"2002-04-26T04:22:01-07:00\" (strings like\n"
"\"2002-04-26T04:22:01\" are also acceptable - duplicity will use the\n"
"current time zone), or ordinary dates like 2/4/1997 or 2001-04-23\n"
"(various combinations are acceptable, but the month always precedes\n"
"the day)."
msgstr "سلسلة زمنية سيئة \"%s\"\n\n"
"السلاسل الزمنية المقبولة هي فترات فاصلة (مثل \"3D64s\"), w3-datetime\n"
"، مثل \"2002-04-26T04:22:01-07:00\" (سلاسل مثل\n"
"\"2002-04-26T04:22:01\" مقبولة أيضاً - الازدواجية سوف تستخدم\n"
"المنطقة الزمنية الحالية)، أو تواريخ عادية مثل 2/4/1997 أو 2001-04-23\n"
"(مجموعات مختلفة مقبولة، ولكن الشهر دائماً يسبق\n"
"اليوم)."

#: ../duplicity/gpginterface.py:694
#, python-brace-format
msgid "GPG process {process.pid} terminated before wait()"
msgstr "انتهت عملية GPG {process.pid} قبل الانتظار()"

#: ../duplicity/lazy.py:349
#, python-format
msgid "Warning: oldindex %s >= newindex %s"
msgstr "تحذير: الفهرس القديم %s >= الفهرس الجديد %s"

#: ../duplicity/lazy.py:424
#, python-format
msgid "Error '%s' processing %s"
msgstr "خطأ '%s' معالجة %s"

#: ../duplicity/lazy.py:436
#, python-format
msgid "Skipping %s because of previous error"
msgstr "تخطي %s بسبب خطأ سابق"

#: ../duplicity/librsync.py:194
msgid "basis_file must be a (true) file or an object whose file attribute is the underlying true file object"
msgstr "basis_file يجب أن يكون ملف (صحيح) أو كائن سمات ملفه هو كائن الملف الحقيقي الأساسي"

#: ../duplicity/manifest.py:95
#, python-format
msgid "Fatal Error: Backup source host has changed.\n"
"Current hostname: %s\n"
"Previous hostname: %s"
msgstr ""

#: ../duplicity/manifest.py:102
msgid "Fatal Error: Backup source directory has changed.\n"
msgstr "خطأ مميت: تم تغيير دليل مصدر النسخ الاحتياطي.\n"

#: ../duplicity/manifest.py:115
msgid "Aborting because you may have accidentally tried to backup two different data sets to the same remote location, or using the same archive directory.  If this is not a mistake, use the --allow-source-mismatch switch to avoid seeing this message"
msgstr "إلغاء لأنه ربما كنت قد حاولت عن طريق الخطأ نسخ نسخة احتياطية لمجموعتين مختلفتين من البيانات إلى نفس الموقع البعيد، أو باستخدام نفس دليل الأرشيف. إذا لم يكن هذا خطأ، استخدم مفتاح --مسموح المصدر - عدم التوافق لتجنب رؤية هذه الرسالة"

#: ../duplicity/manifest.py:221
#, python-format
msgid "Found manifest volume %s"
msgstr "تم العثور على حجم البيان %s"

#: ../duplicity/manifest.py:228
#, python-format
msgid "Found %s volumes in manifest"
msgstr "تم العثور على مجلدات %s في البيان"

#: ../duplicity/manifest.py:249
msgid "Manifest file '{self.fh.base if self.fh else ''}' is corrupt: "
msgstr "الملف '{self.fh.base if self.fh else ''}' تالف: "

#: ../duplicity/manifest.py:268
msgid "Manifests not equal because different volume numbers"
msgstr "البيانات غير متساوية لأن أرقام حجم مختلفة"

#: ../duplicity/manifest.py:273
msgid "Manifests not equal because volume lists differ"
msgstr "البيانات غير متساوية لأن قوائم الصوت تختلف"

#: ../duplicity/manifest.py:277
msgid "Manifests not equal because hosts or directories differ"
msgstr "البيانات غير متساوية لأن المضيفين أو الدلائل مختلفة"

#: ../duplicity/manifest.py:438
msgid "Warning, found extra Volume identifier"
msgstr "تحذير، تم العثور على معرف حجم إضافي"

#: ../duplicity/manifest.py:464
msgid "Other is not VolumeInfo"
msgstr "الأخرى ليست معلومات مستوى الصوت"

#: ../duplicity/manifest.py:467
msgid "Volume numbers don't match"
msgstr "أرقام الصوت غير متطابقة"

#: ../duplicity/manifest.py:470
msgid "start_indicies don't match"
msgstr "بدء_المؤشرات غير متطابقة"

#: ../duplicity/manifest.py:473
msgid "end_index don't match"
msgstr "الرقم القياسي غير متطابق"

#: ../duplicity/manifest.py:478
msgid "Hashes don't match"
msgstr "التجزئة غير متطابقة"

#: ../duplicity/patchdir.py:73 ../duplicity/patchdir.py:80
#, python-format
msgid "Patching %s"
msgstr "تعديل %s"

#: ../duplicity/patchdir.py:548
#, python-format
msgid "Error '%s' patching %s"
msgstr "خطأ '%s' تصحيح %s"

#: ../duplicity/patchdir.py:625
#, python-format
msgid "Writing %s of type %s"
msgstr "كتابة %s من النوع %s"

#: ../duplicity/path.py:109
#, python-format
msgid "Warning: %s invalid devnums (0x%X), treating as (0, 0)."
msgstr "تحذير: %s الديفانوس غير صالح (0x%X)، المعاملة كـ (0، 0)."

#: ../duplicity/path.py:239 ../duplicity/path.py:299
#, python-format
msgid "Warning: %s has negative mtime, treating as 0."
msgstr "تحذير: %s لديه وقت سالب، يعامل ك0."

#: ../duplicity/path.py:372
#, python-format
msgid "New file %s"
msgstr "ملف جديد %s"

#: ../duplicity/path.py:375
#, python-format
msgid "File %s is missing"
msgstr "الملف %s مفقود"

#: ../duplicity/path.py:378
#, python-format
msgid "File %%s has type %s, expected %s"
msgstr "الملف %%يحتوي على نوع %s، المتوقع %s"

#: ../duplicity/path.py:383 ../duplicity/path.py:412
#, python-format
msgid "File %%s has permissions %s, expected %s"
msgstr "الملف %%s لديه أذونات %s، المتوقعة %s"

#: ../duplicity/path.py:389
#, python-format
msgid "File %%s has mtime %s, expected %s"
msgstr "الملف %%س لديه وقت %s، المتوقع %s"

#: ../duplicity/path.py:400
#, python-format
msgid "Data for file %s is different"
msgstr "البيانات للملف %s مختلفة"

#: ../duplicity/path.py:408
#, python-format
msgid "Symlink %%s points to %s, expected %s"
msgstr "رابط رمزي %%s نقطة إلى %s، متوقعة %s"

#: ../duplicity/path.py:415
#, python-format
msgid "Device file %%s has numbers %s, expected %s"
msgstr "ملف الجهاز %%يحتوي على أرقام %s، المتوقع %s"

#: ../duplicity/path.py:605
#, python-format
msgid "Making directory %s"
msgstr "إنشاء الدليل %s"

#: ../duplicity/path.py:615
#, python-format
msgid "Deleting %s"
msgstr "حذف %s"

#: ../duplicity/path.py:624
#, python-format
msgid "Touching %s"
msgstr "اللمس %s"

#: ../duplicity/path.py:632
#, python-format
msgid "Deleting tree %s"
msgstr "حذف شجرة %s"

#: ../duplicity/robust.py:84
#, python-format
msgid "Error listing directory %s"
msgstr "خطأ في إدراج الدليل %s"

#: ../duplicity/selection.py:115
#, python-format
msgid "Skipping socket %s"
msgstr "تخطي المقبس %s"

#: ../duplicity/selection.py:121
#, python-format
msgid "Error initializing file %s"
msgstr "خطأ في تهيئة الملف %s"

#: ../duplicity/selection.py:127 ../duplicity/selection.py:167
#: ../duplicity/selection.py:615
#, python-format
msgid "Error accessing possibly locked file %s"
msgstr "خطأ في الوصول إلى ملف محتمل مقفل %s"

#: ../duplicity/selection.py:182
#, python-format
msgid "Warning: base %s doesn't exist, continuing"
msgstr "تحذير: قاعدة %s غير موجودة، الاستمرار"

#: ../duplicity/selection.py:184 ../duplicity/selection.py:205
#: ../duplicity/selection.py:208
#, python-format
msgid "Selecting %s"
msgstr "اختيار %s"

#: ../duplicity/selection.py:263
#, python-format
msgid "                The last file selection option is the filter option %s, which will have no\n"
"                effect as there are no subsequent file selection options. Exiting because\n"
"                this probably isn't what you meant."
msgstr "                خيار اختيار الملف الأخير هو خيار الفلتر %s، والذي لن يكون له تأثير\n"
"                لأنه لا توجد خيارات اختيار الملفات اللاحقة. الخروج لأن\n"
"                ربما ليس ما تعنيه."

#: ../duplicity/selection.py:278
#, python-format
msgid "                Only these filter mode options were specified:\n"
"                    %s\n"
"                Case sensitive globbing is the default behaviour and so this has no effect.\n"
"                Exiting because this probably isn't what you meant."
msgstr "                فقط خيارات وضع التصفية هذه تم تحديدها:\n"
"                    %s\n"
"                وعولمة الحالة حساسة هو السلوك الافتراضي وبالتالي ليس لهذا أي تأثير.\n"
"                الخروج لأن هذا ربما ليس ما تعنيه."

#: ../duplicity/selection.py:346
#, python-format
msgid "                Fatal Error: The file specification\n"
"                    %s\n"
"                cannot match any files in the base directory\n"
"                    %s\n"
"                Useful file specifications begin with the base directory or some\n"
"                pattern (such as '**') which matches the base directory."
msgstr "                خطأ مميت: لا يمكن لمواصفات الملف\n"
"                    %s\n"
"                أن تتطابق مع أي ملفات في الدليل الأساسي\n"
"                    %s\n"
"                من مواصفات الملفات المفيدة تبدأ بالدليل الأساسي أو بنمط\n"
"                (مثل '**') الذي يطابق الدليل الأساسي."

#: ../duplicity/selection.py:360
#, python-format
msgid "Fatal Error while processing expression\n"
"%s"
msgstr "خطأ مميت أثناء معالجة التعبير\n"
"%s"

#: ../duplicity/selection.py:378
#, python-format
msgid "Reading files to backup from list %s"
msgstr "قراءة الملفات للنسخ الاحتياطي من القائمة %s"

#: ../duplicity/selection.py:403
#, python-format
msgid "                Files-from list contains the absolute path:\n"
"                    %s\n"
"                All paths specified in a files-from list must be given relative to the backup\n"
"                source path."
msgstr "                الملفات من القائمة تحتوي على المسار المطلق:\n"
"                    %s\n"
"                يجب إعطاء جميع المسارات المحددة في الملفات من القائمة بالنسبة لمسار المصدر\n"
"."

#: ../duplicity/selection.py:418
msgid "                Files-from list specified which contains no files, the backup will be empty as\n"
"                a result. Exiting as this probably isn't what you meant,"
msgstr ""

#: ../duplicity/selection.py:435
#, python-format
msgid "                Last selection expression:\n"
"                    %s\n"
"                only specifies that files be included.  Because the default is to\n"
"                include all files, the expression is redundant.  Exiting because this\n"
"                probably isn't what you meant."
msgstr "                تعبير الاختيار الأخير:\n"
"                    %s\n"
"                يحدد فقط أن الملفات يجب أن تكون مشمولة. لأن الافتراضي هو\n"
"                يتضمن جميع الملفات، التعبير غير ضروري. الخروج لأن هذا\n"
"                ربما ليس ما تعنيه."

#: ../duplicity/selection.py:497
#, python-format
msgid "Reading %s filelist %s"
msgstr "قراءة %s ملفخ %s"

#: ../duplicity/selection.py:538
#, python-format
msgid "Error compiling regular expression %s"
msgstr "خطأ في تجميع التعبير العادي %s"

#: ../duplicity/tempdir.py:157
#, python-format
msgid "Using temporary directory %s"
msgstr "استخدام الدليل المؤقت %s"

#: ../duplicity/tempdir.py:201
#, python-format
msgid "Registering (mktemp) temporary file %s"
msgstr "تسجيل (ملف) مؤقت %s"

#: ../duplicity/tempdir.py:227
#, python-format
msgid "Registering (mkstemp) temporary file %s"
msgstr "تسجيل ملف (ملف) مؤقت %s"

#: ../duplicity/tempdir.py:259
#, python-format
msgid "Forgetting temporary file %s"
msgstr "نسيان الملف المؤقت %s"

#: ../duplicity/tempdir.py:262
#, python-format
msgid "Attempt to forget unknown tempfile %s - this is probably a bug."
msgstr "محاولة لنسيان الملف الإفتراضي %s غير معروف، ربما يكون هذا خطأ."

#: ../duplicity/tempdir.py:281
#, python-format
msgid "Removing still remembered temporary file %s"
msgstr "إزالة ما زال يتذكر الملف المؤقت %s"

#: ../duplicity/tempdir.py:284
#, python-format
msgid "Cleanup of temporary file %s failed"
msgstr "فشل تنظيف الملف المؤقت %s"

#: ../duplicity/tempdir.py:290
#, python-format
msgid "Cleanup of temporary directory %s failed - this is probably a bug."
msgstr "فشل تنظيف الدليل المؤقت %s - ربما هذا خطأ."

#: ../duplicity/util.py:176
#, python-format
msgid "Acquiring lockfile %s"
msgstr "الحصول على ملف قفل %s"

#: ../duplicity/util.py:191
#, python-format
msgid "Releasing lockfile %s"
msgstr "إطلاق ملف قفل %s"

